### PR TITLE
alacritty: update to 0.13.1

### PR DESCRIPTION
**Pull request by Daxot**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by tjkirch** [comment]
@Daxot @heindsight Was this intended to update Chrome too?  I was going to give it a test, but I don't use Chrome, and updating it here seemed odd.

**Comment by Daxot** [comment]
> @Daxot @heindsight Was this intended to update Chrome too? I was going to give it a test, but I don't use Chrome, and updating it here seemed odd.

Oh, my fault. Resolved!

**Comment by tjkirch** [comment]
Thanks!  I can confirm that this update works for me.

However, I quickly ran into https://github.com/alacritty/alacritty/issues/7472 (a startup performance problem) so I downgraded back to 0.12.3.  It may be worth waiting for 0.13.1?  A number of people with different setups reported the same problem in that issue.

**Comment by dataCobra** [comment]
@tjkirch did you had the time to check if 0.13.1 fixed your startup issue?

**Comment by tjkirch** [comment]
> @tjkirch did you had the time to check if 0.13.1 fixed your startup issue?

Yes, 0.13.1 works fine for me, and it no longer has the startup delay.  I'll make a suggestion to the PR.

(Tangent: I had to update `.xbps-checkvers-0.58.plist` to reference cargo 1.75 instead of 1.74, otherwise it was going to rebuild all of rust, llvm, etc., which my machine can't handle.  I'm not an expert with xbps-based builds, so I don't know if that would affect anything with this package; I'm guessing it's just a timing issue in the repos...)

File: srcpkgs/alacritty/template [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Template file for 'alacritty'
 pkgname=alacritty
-version=0.12.3
+version=0.13.0
```
Comment (tjkirch) [inline srcpkgs/alacritty/template:3]:
```suggestion
version=0.13.1
```

File: srcpkgs/alacritty/template [hunk 2]
```diff
@@ -13,7 +13,7 @@ license="Apache-2.0"
 homepage="https://github.com/alacritty/alacritty"
 changelog="https://raw.githubusercontent.com/alacritty/alacritty/master/CHANGELOG.md"
 distfiles="https://github.com/alacritty/alacritty/archive/refs/tags/v${version}.tar.gz"
-checksum=7825639d971e561b2ea3cc41e30b57cde8e185a400fee001843bb634df6b28ab
+checksum=1c1cebf20e10bb26dc8bc735bb2d02eb88df89180d6c59f5d946a0a1d3d585b2
```
Comment (tjkirch) [inline srcpkgs/alacritty/template:16]:
```suggestion
checksum=38a42e23e1e6faaa9e300347df3f7b58b6182908a701517aac6e296fbdf37451
```

**Review by tjkirch — COMMENTED** [review]
I tested 0.13.1 using these changes, and it worked fine and no longer had the startup delay from 0.13.0.

**Comment by Daxot** [comment]
Updated PR to 0.13.1 version

**Review by tjkirch — APPROVED** [review]
Thanks!

**Comment by 0323pin** [comment]
`alacritty` is broken and I can not launch a terminal from my user session. I've installed `xterm` to get the reason, it reads:

```
thread 'main' panicked at /host/cargo/registry/src/index.crates.io-6f17d22bba15001f/xkbcommon-dl-0.4.1/src/x11.rs:59:28:
Library libxkbcommon-x11.so could not be loaded.
```

I run `alacritty-dev-0.14.0` on NetBSD and suspect there as a dependency or, linking missing in the updated Void package.
Can someone have a look?
Thanks!

**Comment by dataCobra** [comment]
Hmm, for me it works flawless.

@0323pin could you please create an issue with all the relevant information?
I think that is better suited in the case then using the merged PR.

**Comment by 0323pin** [comment]
Sure, I'll create an issue.

**Comment by dataCobra** [comment]
Thanks but I can tell you I've already found the issue.

alacritty has a dependency with libxkbcommon-x11.

I'll create a PR asap to get that fixed.

**Comment by 0323pin** [comment]
Thanks!

Yes, I know about the dependency. I maintain the package for NetBSD and as said run the dev version also.

**Comment by 0323pin** [comment]
Manually installing the dependency fixes it.

**Comment by dataCobra** [comment]
@0323pin I created a new PR #48197 for the fix and will contact the Void devs for a quick merge.

**Comment by 0323pin** [comment]
Thank you.

**Pull request merged**